src/gravel/controllers/nodes/mgr.py

```diff
@@ -326,6 +333,20 @@ def gen() -> str:
 
         self._load()
 
+    async def _finish_bootstrap_config(self) -> None:
+        mon: Mon = Mon()
+        try:
+            mon.set_allow_pool_size_one()
+        except CephCommandError as e:
+            logger.error("=> mgr -- unable to allow pool size 1")
+            logger.debug(str(e))
```

Comment:
(If you use `logger.exception` the logger will print the traceback along with the error string etc instead of needing to manually log it (but I realise you're using the debug level so this might not be helpful here)).

Reply:
huh - had no idea. That's really helpful. Although in this case I think we might just want the reason and not the whole stack trace.